### ISSUE TITLE
From where can I get /dist directory?

### ISSUE THREAD

**Issue by gorgotluk** [hakimel/Ladda#84]
Hi, 
From where can I get dist directory with compiled version of ladda? I want use it in my PHP application but I can't /dist directory with newest version.

**Comment by theodorejb** [comment]
Run `npm install ladda`. The compiled package with a /dist folder will be put in node_modules/ladda.

**Comment by gorgotluk** [comment]
I try but I get certificate Error.

**Comment by theodorejb** [comment]
Have you tried googling the error to see what might be causing it? It doesn't sound like an issue related to Ladda.

**Comment by gorgotluk** [comment]
I don't Say it is ladda issue but I want to know  that is other Way to get compiled ladda version

**Comment by adonespitogo** [comment]
I didn't get any error but only css are in /dist folder and no js files

**Comment by theodorejb** [comment]
@adonespitogo That's expected. The package is written in standard JavaScript, so these files don't need to be compiled, only bundled. You should be able to import the ES6 module in your project and bundle everything together with Webpack or Rollup.

**Comment by adonespitogo** [comment]
@theodorejb thanks for the quick response, but time is critical for me. I just downloaded the files from the demo site. Could have much convenient if it was available as a drop-in file without needing ES6 bundler. I'm using gulp

**Comment by theodorejb** [comment]
You could also stick with version 1.x if you don't want to use a bundler. I believe that's the version that's still on the demo site.

**Comment by sbwMikael** [comment]
This is very frustrating. I'm not directing my ire at Ladda specifically, but the trend for not providing deployable scripts is pretty darned irritating. I've avoided grappling with any of this stuff until now - NodeJS, NPM, Grunt, and all of these other names that are opaque to me - because I've managed happily without them and I don't have time to learn yet another package management system right now.

So I fired up a Linux Mint install in a virtual machine, went through installing NodeJS, followed a tutorial online about setting things up, ran all of the various commands, and hit the "SyntaxError: Unexpected token import".

So I go to look on StackOverflow, and it seems that NodeJS "doesn't understand ES6 syntax", and there are messages about adding yet another application called Babel. Except, that's been recent refactored in version 6 and now involved pages of commands to get things working as expected. All I want is one JS file! Argh.
